tests/integration/pilot/vm/testdata/OS_Support_List

```diff
@@ -0,0 +1,6 @@
+app_sidecar_trusty
+app_sidecar_xenial
+app_sidecar_focal
+app_sidecar_debian8
+app_sidecar_debian9
+app_sidecar_debian10
```

Comment:
the test won't pass for now since these docker image's current do not exist in the istio-testing gcr. Will re-run the test after I push the images to the gcr.

Reply:
We need to build these as part of the test, not push them to GCR. That way we are actually testing the code we are changing in the PR. If you just have static images we are using then we are just testing the same code over and over again - even if the PR broke something the test would pass.

Reply:
Understood. That makes sense. It's just that I saw `app_sidecar` is pushed to the GCR, so I assumed we would need to push the images to GCR. I will try to build images on the fly and make the changes. Thanks for your comment!

Reply:
we build it on the fly during tests, then additionally publish to GCR *after* a PR is merged. See prow/lib.sh where we do the building

Reply:
Updated the docker file and prow/lib.sh to build image on the fly.